examples/lightgbm_link_baseline.py

```diff
@@ -86,9 +86,13 @@
 
 # randomly subsample in case training data size is too large.
 sampled_train_table = copy.deepcopy(train_table)
+sampled_val_table = copy.deepcopy(val_table)
 if args.sample_size > 0 and args.sample_size < len(sampled_train_table):
     sampled_idx = np.random.permutation(len(sampled_train_table))[: args.sample_size]
     sampled_train_table.df = sampled_train_table.df.iloc[sampled_idx]
+if args.sample_size > 0 and args.sample_size < len(sampled_val_table):
+    sampled_idx = np.random.permutation(len(sampled_val_table))[: args.sample_size]
+    sampled_val_table.df = sampled_val_table.df.iloc[sampled_idx]
```

Comment:
I don't want validation data to be sampled because we report validation performance in our paper.
Also, given that we use full test data anyway, the speedup is not that large.

Reply:
I see, reverting this now.